lib/screens/contribution/contribution_screen.dart

```diff
@@ -0,0 +1,214 @@
+          child: FittedBox(
+            fit: BoxFit.cover,
+            child: Image.memory(widget.contribution.org.logo),
+          ),
+        ),
+        const SizedBox(width: 15),
+        Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Text(
+              'to',
+              style: Theme.of(context)
+                  .textTheme
+                  .bodySmall!
+                  .copyWith(color: COLOR_GRAY),
+            ),
+            Text(
+              widget.contribution.org.name,
+              style: const TextStyle(fontWeight: FontWeight.bold),
+            ),
+            Text(
+              '@${widget.contribution.org.username}',
+              style: Theme.of(context).textTheme.labelMedium!.copyWith(
+                    color: COLOR_GRAY,
+                    fontWeight: FontWeight.w500,
+                  ),
+            ),
+          ],
+        ),
+      ],
+    );
+  }
+
+  buildMainSection(BuildContext context) {
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        RichText(
+          text: const TextSpan(
+            children: [
+              WidgetSpan(child: SizedBox(width: 50)),
+              TextSpan(text: 'Your future is created'),
+              TextSpan(text: '\nby what you do today'),
+              WidgetSpan(child: SizedBox(width: 50)),
+            ],
+          ),
+        ),
+        const SizedBox(height: 10),
+        ConstrainedBox(
+          constraints: const BoxConstraints(maxWidth: 300),
+          child: Image.asset(
+            'assets/images/contribution_img.png',
+            width: MediaQuery.of(context).size.width * 0.7,
+          ),
+        ),
+      ],
+    );
+  }
+
+  navigateToDetails(Contribution contribution) {
+    if (context.mounted) {
+      Navigator.of(context).pushAndRemoveUntil(
+        MaterialPageRoute(
+          builder: (_) => ContributionDetailsScreen(contribution: contribution),
+        ),
+        (route) => false,
+      );
+    }
+  }
+
+  handleStopContributionPressed(BuildContext context) async {
+    setState(() {
+      isLoading = true;
+    });
+    try {
+      final response = await orgsApi.stopContribution(
+        widget.contribution.org.id,
+        widget.contribution.id!,
+      );
+      final stoppedContribution = Contribution.fromJson(response.data);
+      navigateToDetails(stoppedContribution);
+    } on DioError catch (error) {
+      print(error);
+      if (error.response!.statusCode == HttpStatus.forbidden) {
+        final stoppedContribution =
+            Contribution.fromJson(error.response!.data['contribution']);
+        navigateToDetails(stoppedContribution);
+      }
+    } catch (error) {
+      print(error);
+    } finally {
+      setState(() {
+        isLoading = false;
+      });
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
```

Comment:
тут тоже можно юзать
```suggestion
      child: ScreenScaffold(
``` ScreenScaffold